.jscsrc

```diff
@@ -0,0 +1,3 @@
+{
+	"preset": "google"
```

Comment:
I picked google, but there are many others to choose from:

http://jscs.info/overview.html#presets